api-examples/rbacv1/main.py

```diff
@@ -0,0 +1,324 @@
+# !/usr/bin/env python3
+import csv
+import os
+import urllib
+
+import requests
+
+SHIFTLEFT_ORG_ID = os.getenv("SHIFTLEFT_ORG_ID")
+SHIFTLEFT_ACCESS_TOKEN = os.getenv("SHIFTLEFT_ACCESS_TOKEN")
+
+API_V4_BASE_URL = "https://www.shiftleft.io/api/v4/"
+API_V4_ORG_PATH = "orgs/{organization_id}/"
+
+
+class SLAPIError:
+    """
+    SLAPIError represents API error details returned by SL API v4
+    """
+
+    def __init__(self, ok=False, code=0, message="", validation_errors=[]):
```

Comment:
```suggestion
    def __init__(self, ok=False, code=0, message="", validation_errors=()):
```
make that immutable or it _will_ bite you